packages/core/src/types/redocly-yaml.ts

```diff
@@ -298,6 +298,7 @@ const AssertionDefinitionAssertions: NodeType = {
         'MACRO_CASE',
         'COBOL-CASE',
         'flatcase',
+        'Sentencecase'
```

Comment:
should we rename it to `Sentence case` (with a space in it)?

I also wander if we should add `Title Case` too for symmetry.